packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -314,6 +314,10 @@ class Tooltip extends React.Component {
           placement={placement}
           anchorEl={this.childrenRef}
           open={open}
+          onMouseOver={childrenProps.onMouseOver}
+          onMouseLeave={childrenProps.onMouseLeave}
+          onFocus={childrenProps.onFocus}
```

Comment:
Do we need the focus/blur listeners?

Reply:
It would allow keyboard users to tap to a button inside the tooltip - I guess that’s rather an edge case but adding a little bit accessibility here will not cause any other side effects

Reply:
Maybe like the drop down nesting example here : https://atomiks.github.io/tippyjs/

Reply:
But the idea is the same - if you manage to tab over fast enough the tooltip will stay open - if you then tab further the tooltip should close